packages/protocol-kit/tests/e2e/utilsContracts.test.ts

```diff
@@ -453,5 +454,41 @@ describe('Contract utils', () => {
       chai.expect(secondPredictedSafeAddress).to.be.equal(expectedSafeAddress)
       chai.expect(thirdPredictedSafeAddress).to.be.equal(expectedSafeAddress)
     })
+
+    itif(safeVersionDeployed > '1.0.0')(
```

Comment:
Beautiful test. Just checking this check is done because in case `safeDeploymentConfig` is not passed, the default version will be (currently) `v1.3.0` and its use would break with the `v1.0.0` contracts deployed by Hardhat when using `customContracts` variable

Reply:
when calling `setup` method in the contract, because of the number or parameters of the function

Reply:
Okay, what I was thinking. All good then!